source/_integrations/zwave_js.markdown

```diff
@@ -60,6 +62,126 @@ Some devices can be auto-discovered, which can simplify the Z-Wave JS setup proc
 
 Additional devices may be discoverable, however only devices that have been confirmed discoverable are listed above.
 
+**700 series USB sticks are fully supported by Z-Wave JS and are recommended for new users purchasing their first USB stick**, but those USB sticks are not presently auto-discoverable by the integration.
+
+## Migrating from previous Z-Wave implementations
```

Comment:
I think we should move the migration and advanced installation sections back down. It's not something that users will need to use more than once, for the most part.

Reply:
I debated that, but users don't notice them at the bottom as it's contrary to how documentation is usually organized.

Reply:
We already have a link to the advanced instructions in the quickstart section at the top. We could add a link to migration there. Something like:

```
If you're migrating from one of the deprecated Z-Wave integrations, `zwave` or `ozw`, please see the [migration section](#migrating-from-previous-z-wave-implementations).
```

Reply:
I know that we do, I'm just saying that I've seen dozens of people say they looked at that and they didn't know the instructions were there, or they didn't see part of them. So what's there isn't terribly effective... I think it's actually made worse that the quick start instructions are at the top and then they skip the link. 

What if we move all the installation instructions down and have a link to those at the top? That way they aren't enticed to just look at the ones that are there?

Reply:
We want to promote the simple installation on HA OS with official add-on. It's ok if the advanced instructions is a bit less prominent.

If I google for "Home Assistant zwave js migration" I can link directly into the migration section from the top google result. It doesn't seem hard to find for me.

Reply:
If you want I'll stet back the move and suggest other changes to the quick start section. That section isn't complete. It doesn't make clear they need one addon and the integration, as an example, which is a common question. Nor does it mention the migration wizard leading to frequent questions whether one exists (that's not really an advanced installation in any event). 

Nor are the requirements clear at the top of the integration page...that's not really part of the advanced installation instructions either. 

I'd expect the documentation to be:
- Requirements
- Installation
- Use

If we're not going to follow that standard format then I think we need a roadmap at the top pointing to those sections.

Reply:
The goal with the integration is to allow users running HA OS a plug and play experience where they can plug in the USB stick, go to the integrations page, and easily set up the integration with a couple of clicks. This is what the quick start section reflects. The user shouldn't need to know more at that point. It also mentions the advanced instructions if the user isn't running HA OS or wants to read and learn more.

Reply:
I understand the goal and intent. Part of an easy user experience is being able to find an answer when you have a question, though. While the intent is for it to be one click lots of people know and have heard there is more than one option. They then have difficulty understanding what's required. Even without zwavejs2mqtt users are confused and frequently install the addon and no integration. I'm just saying let's add a short section at the top that says:

- If you want to run zwavejs, you need a usb stick (link to requirements) and to install the integration, which will then install (link to configuration section) the zwavejs addon automatically. 

- To instead use the zwavejs2mqtt addon with the integration, see the advanced installation instructions (link to advanced instructions).


Then remove the duplication. If configuration is going to be at the top, then the advanced instructions should link to there and say "Continue with configuration".

Reply:
In fact, what if we cut everything above Configuration and add the links to migration and advanced under that. Then get rid of Installing and Configuring under advanced. And move Discovery via USB to an FAQ question? That would go a long way...

Reply:
Your last suggestion sounds good!

Reply:
Ok. I'll make those changes later and push them.

Reply:
How about these changes?